lib/client-assets.php

```diff
@@ -133,6 +133,224 @@ function gutenberg_override_style( $handle, $src, $deps = array(), $ver = false,
 	wp_register_style( $handle, $src, $deps, $ver, $media );
 }
 
+/**
+ * Registers all the WordPress packages scripts that are in the standardized
+ * `build/` location.
+ *
+ * @since 4.3.0
+ */
+function gutenberg_register_packages_scripts() {
+	$packages_dependencies = array(
+		'api-fetch' => array( 'wp-polyfill', 'wp-hooks', 'wp-i18n', 'wp-url' ),
```

Comment:
@aduth - could we assume that all scripts require `wp-polyfill` and inject it in the loop?

Reply:
We can assume for all Gutenberg-related core scripts, yes.

Reply:
Awesome, I took that into account 👍

Reply:
There's a bit of me which is sad in losing the explicitness, at least in considering `package-dependencies.php` as an exhaustive set. I suppose it is more an implicit thing if we're making a blanket statement that _all_ new scripts should define `wp-polyfill` as a dependency.